sui/pause.js

```diff
@@ -0,0 +1,243 @@
+const { Command, Option } = require('commander');
+const { TxBuilder } = require('@axelar-network/axelar-cgp-sui');
+const { loadConfig, saveConfig, getChainConfig, printInfo } = require('../common/utils');
+const {
+    addBaseOptions,
+    addOptionsToCommands,
+    getWallet,
+    printWalletInfo,
+    broadcastFromTxBuilder,
+    getAllowedFunctions,
+} = require('./utils');
+
+const SPECIAL_PAUSE_FUNCTION_TAGS = {
+    ALL: 'all', // All EVM chains that have InterchainTokenService deployed
+    DEFAULT: 'default',
+};
+
+const SPECIAL_UNPAUSE_FUNCTION_TAGS = {
+    DISALLOWED: 'disallowed', // All EVM chains that have InterchainTokenService deployed
+    DEFAULT: 'default',
+};
+
+function getVaraiblesForPackage(chain, packageName) {
+    if (packageName === 'AxelarGateway') {
+        const contractConfig = chain.contracts.AxelarGateway;
+        return {
+            packageId: contractConfig.address,
+            singletonId: contractConfig.objects.Gateway,
+            versionedId: contractConfig.objects.Gatewayv0,
+            ownerCapId: contractConfig.objects.OwnerCap,
+            moduleName: 'gateway',
+            defaultFunctions: {
+                versions: [0, 0],
```

Comment:
the current package version should be read from the contract config

Reply:
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/testnet.json#L2354